week-4/D-methods/exercise-4.js

```diff
@@ -0,0 +1,23 @@
+/* 
+Alice has a list of good friends. Define a method "makeFriend" to add a new friend called "Bob" to her list.
```

Comment:
Maybe drop the 'called "Bob"' here, 'cos that's a specific example. A few students are still confused about the difference between adding functions with abstract parameters and specifically implementing them with the test data hard-coded :).